### PR TITLE
Initial command support from cabal-client

### PR DESCRIPTION
**Pull request by nikolaiwarner**
This switches cabal desktop to use the command processing built into `cabal-client`.

Also did some style improvements in preparation for context menus.

### CODE REVIEW

**Comment by cblgh** [comment]
i'll read through it during lunch!

but one thing that would be really nice is if we manage to merge https://github.com/cabal-club/cabal-client/pull/38/files first, and then rely on that version of cabal-client. that would give us the beginnings of moderation capabilities in desktop, too!

File: app/actions.js [hunk 1]
```diff
@@ -23,7 +22,25 @@ const client = new Client({
...
+        }
+        addStatusMessage({ addr: cabal.key, text: helpContent })
+      }
+    }
+  }  
+})
+// Disable a few slash commands for now
+const removedCommands = ['add', 'channels', 'clear', 'ids', 'names', 'new', 'qr', 'whoami', 'whois']
+removedCommands.forEach((command) => {
+  client.removeCommand(command)
```
Comment (cblgh) [inline app/actions.js:43]: ➕

Comment (nikolaiwarner) [reply app/actions.js:43]: yep those commands need a signal from cabal-client that they're happening and we can add them back in when we can

**Comment by nikolaiwarner** [comment]
Thanks @cblgh ! Yep, this was just an initial setup to get the core of things working and stable. I'll bump cabal-client in another pr where i add in the context menus :D

**Pull request merged**